apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -312,11 +317,11 @@ export class ChangeAction extends BaseRushAction {
   }
 
   private _getChangedPackageNames(): string[] {
-    const changedFolders: (string | undefined)[] | undefined = VersionControl.getChangedFolders(
+    const changedFiles: string[] = VersionControl.getChangedFiles(
```

Comment:
Switched to `getChangedFiles` since ignore patterns might apply to files or folders (e.g. `__tests__` or `**/*.test.ts`).

I'm not sure why the return type was `(string | undefined)[] | undefined` given `VersionControl.getChangedFolders` and `VersionControl.getChangedFiles` _always_ return an array of strings (it's just empty if there are no changes). So I updated it to `string[]`